src/mmt-odk/src/info/kwarc/mmt/odk/SCSCP/Example/ClientExample.scala

```diff
@@ -9,7 +9,7 @@ object ClientExample {
     import info.kwarc.mmt.odk.OpenMath._
 
     // establish a connection
-    val client = SCSCPClient("scscp.gap-system.org")
+    val client = SCSCPClient("localhost")
```

Comment:
Not sure if it is the best idea to switch this to `localhost`. The intention behind having `scscp.gap-system.org` is that the example can be run without having to setup a server on your own.